pkg/volume/csi/csi_mounter.go

```diff
@@ -121,6 +160,12 @@ func (c *csiMountMgr) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 
+	if err := os.MkdirAll(dir, 0750); err != nil {
```

Comment:
If mount fails after this point, we should cleanup this directory before returning.